packages/teleport-plugin-react-proptypes/src/index.ts

```diff
@@ -65,7 +75,7 @@ export const createPlugin: ComponentPluginFactory<ReactJSPropTypesConfig> = (con
     })
 
     // push export of component after declarations of types
-    exportChunk.linkAfter.push(typesOfPropsChunkName, defaultPropsChunkName)
+    exportChunk.linkAfter.push(typesOfPropsChunkName)
```

Comment:
will this invert the order? default props coming before types? I would keep the original order as this is how components usually are defined, first propTypes, then defaultProps

Reply:
I think the order is fine with the UIDL's I tested, but let me recheck few more 👍